.github/workflows/python-package.yml

```diff
@@ -31,13 +31,13 @@ jobs:
 
     - name: Create tasks needed for testing
       run: |
-          python scripts/tasks_retrieval/gene2gene_task_creation.py --allow-downloads True
-          python scripts/tasks_retrieval/Genecorpus_tasks_creation.py --allow-downloads True
-          python scripts/tasks_retrieval/HLA_task_creation.py --allow-downloads True
-          python scripts/tasks_retrieval/HPA_tasks_creation.py --allow-downloads True
-          python scripts/tasks_retrieval/humantfs_task_creation.py --allow-downloads True
-          python scripts/tasks_retrieval/Reactome_tasks_creation.py  --allow-downloads True
-          python scripts/tasks_retrieval/uniprot_keyword_tasks_creation.py  --allow-downloads True
+          python scripts/tasks_retrieval/gene2gene_task_creation.py --allow-downloads True || true
```

Comment:
please only add the `|| true` to the tasks for which the url is not stable. Also, please add an issue to correct the url and resolve this problem long-term. Most likely by NOT downloading automatically